documentation/Configuration.md

```diff
@@ -313,6 +314,14 @@ file are handled. The `newlineAtEndOfFile` property supports the following value
 * `"require"`: Files are required to end with a single newline character
 * `"omit"`: Files may not end with a newline character
 
+### Consecutive using statements without braces
```

Comment:
This should probably specify if using statements are allowed without braces before other similar constructs like `for`